### PR TITLE
Adding diagrams

### PR DESCRIPTION
**Pull request by luizaandrade**

### CODE REVIEW

**Comment by bbdaniels** [comment]
These are so cool! Just a few general notes on touchup to get them over the finish line. Fonts, colors, and alignment need some tiny detail work; and the verb tense of all the items needs to be brought into agreement. I put specific comments in the markup attached. Let me know if any questions!
[Figures BBD.pdf](https://github.com/worldbank/dime-data-handbook/files/5105430/Figures.BBD.pdf)

File: .gitignore [hunk 1]
```diff
@@ -59,6 +59,7 @@
 # More
 !/**/*.sh
 !/**/*.bib
+!/**/*.png
```
Comment (kbjarkefur) [inline .gitignore:62]:
```suggestion
!/diagrams/*.png
```

**Comment by kbjarkefur** [comment]
This PR is starting to have conflicts. we should start this over after `mj-review` is merged. would it be a lot of work to do this again? If so we can figure this out without starting over

**Comment by bbdaniels** [comment]
The conflicts that are there are really minor, just changes to wording in two obvious places. However @luizaandrade even though I made comments on the diagrams I am happy to resolve the current versions and the code merged in now, it's a lot easier to update the images with no conflicts in another branch right, plus we can see them render?

**Comment by bbdaniels** [comment]
In #461 now.

**Pull request merged**